src/tar.ts

```diff
@@ -39,7 +22,7 @@ async function getTarPath(args: string[]): Promise<string> {
 
 async function execTar(args: string[], cwd?: string): Promise<void> {
     try {
-        await exec(`"${await getTarPath(args)}"`, args, { cwd: cwd });
+        await exec(`${await getTarPath(args)}`, args, { cwd: cwd });
```

Comment:
Is there a reason you're removing these quotes around the tar path?

On Windows, the tar path might have spaces in it and the quotes are needed make sure it's a valid path to exec

Reply:
Forgot about Windows. Reverted.